clang/test/CXX/dcl.decl/dcl.meaning/dcl.fct/p13.cpp

```diff
@@ -58,8 +58,9 @@ void b(T[] ...);
 
 template<typename T>
 void c(T ... []); // expected-error {{expected expression}} \
```

Comment:
To be clear, it's not a "pack-index-specifier" even in C++26: it doesn't match the grammar because there's no constant-expression, and the constant-expression is not optional in the pack-index-specifier grammar.